### ISSUE TITLE
Убрать мгновенный забор крови шприцом [Proposal]

### ISSUE THREAD

**Issue by F0xiq** [TauCetiStation/TauCetiClassic#12165]
#### Подробное описание проблемы
Мгновенный забор крови шприцом, крайне нелогично, особенно с учетом того, что инъекции даже самому себе тратят время.
#### Что должно было произойти
Забор крови возможно даже более сложная операция, чем инъекция и должна затрачивать время.
#### Что произошло на самом деле
Кровь мгновенно набирается в шприц

**Comment by Deahaka** [comment]
Надо на все действия просто повесить дуафтер по 5 секунд минимум тогда будет реалистично

**Comment by Deahaka** [comment]
Но в качестве защиты генокрадов от проверок может это и адекватно. Только об этом 0 слов в ишуе

**Comment by TgT4** [comment]
> Надо на все действия просто повесить дуафтер по 5 секунд минимум тогда будет реалистично
на импланты тоже, пока кнопку активации ищешь в заднем проходе?

**Comment by RONSI7** [comment]
> Но в качестве защиты генокрадов от проверок может это и адекватно. Только об этом 0 слов в ишуе

Точно. Когда к тебе молча подбегает кто-то из СБ и мгновенно берет у тебя кровь, а ты генокрад. В такие моменты говно начинает бурлить знатно.
